.github/bin/s3/delete-s3-bucket.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -uo pipefail
+
+S3_SCRIPTS_DIR="${BASH_SOURCE%/*}"
+
+if [[ ! -f "${S3_SCRIPTS_DIR}/.bucket-identifier" ]];  then
+    echo "Missing file ${S3_SCRIPTS_DIR}/.bucket-identifier"
+    exit 0
```

Comment:
The message is error-like and so should go to stderr

Document why exit 0 (success), despite this looking like error condition

Reply:
Changed to exit 1